src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +32,36 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
```

Comment:
Loader should be rendered inside TodoModal to pass the tests

Reply:
How to do that sir? Does it mean the need to put a loader  Inside the .modal-card div?